Source/WebKit/UIProcess/API/haiku/WebView.h

```diff
@@ -36,7 +36,7 @@ class BWebView
 {
     public:
         BWebView(BRect, BWindow*);
-        void initializeOnce();
+        void initialize();
```

Comment:
Ah, I should revert it to initializeOnce then.

It doesn't work in the constructor because... hold, on. Actually, it does with the new implementation.